kv/dist_sender_server_test.go

```diff
@@ -250,11 +250,13 @@ func TestMultiRangeScanReverseScanInconsistent(t *testing.T) {
 	}
 	sr := reply.(*roachpb.ScanResponse)
 
-	if l := len(sr.Rows); l != 1 {
-		t.Fatalf("expected 1 row; got %d", l)
+	if l := len(sr.Rows); l != len(keys) {
+		t.Fatalf("expected %d rows; got %d", len(keys), l)
 	}
-	if key := string(sr.Rows[0].Key); keys[0] != key {
-		t.Errorf("expected key %q; got %q", keys[0], key)
+	for i := range sr.Rows {
+		if key := string(sr.Rows[i].Key); keys[i] != key {
+			t.Errorf("expected key %q; got %q", keys[i], key)
+		}
```

Comment:
This seems to have been a bug in the previous test. AFAICT, we should get 2 rows here and below, but previously (without the new `WaitHealthy()` time) we were getting only 1. Am I missing anything?